SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -230,6 +232,28 @@ void USpatialGameInstance::HandleOnConnected()
 #endif
 
 	OnSpatialConnected.Broadcast();
+
+	const UWorld* World = GetWorld();
+	if (World != nullptr)
```

Comment:
nit - declare world inside if statement per the unreal gdk codelines - ie. `if (const UWorld* World = GetWorld())`